instruments/astrooda_antares/astrooda_antares.js

```diff
@@ -0,0 +1,38 @@
+jQuery(document).ready(function($){
```

Comment:
thanks cool! Need to give Mohamed to review, let's try to agree on this flow on Monday.

Reply:
This is just an attempt at copy-paste programming
The validation still doesn't work